salt/modules/vagrant.py

```diff
@@ -597,7 +597,12 @@ def get_ssh_config(name, network_mask="", get_private_key=False):
             "{User}@{HostName} ifconfig".format(**ssh_config)
         )
 
-        log.info("Trying ssh -p {Port} {User}@{HostName} ifconfig".format(**ssh_config))
+        log.info(
+            "Trying ssh -p %s %s@%s ifconfig",
```

Comment:
**suggestion** this one is a bit less clear this way. Is `'Trying ssh -p %(Port)s %(User)s@(HostName)s ifconfig'` better? :thinking:  Then it would still just be `log.info(..., ssh_config)` :shrug:

Reply:
Agreed.